cluster-state-service/cmd/root.go

```diff
@@ -46,6 +47,7 @@ a localized data store, which provides you a near-real-time view of your cluster
 	}
 	// TODO: Fix the description
 	rootCmd.PersistentFlags().StringVar(&config.SQSQueueName, sqsQueueNameFlag, "", "SQS queue name")
+	rootCmd.PersistentFlags().StringVar(&config.KinesisStreamName, kinesisStreamNameFlag, "", "Kinesis stream name")
```

Comment:
I prefer the URI style approach. I'll go this direction. It also sounds like this is a change that you'd like to accept given it's improved, so I'll add tests and update the pull request. Should I flatten my commits as I go, or only at the end when you're ready to merge?  Not sure the preferred style for this repo.